seeds/dashboards/pevr/schema.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+
+seeds:
+  - name: common_indicateurs_pevr
+    description: Mapping des indicateurs communs avec leur description, leur code_matière, si c'est le cas, et leur numéros de compétences.   
+    config:
+      tags: 
+        - pevr
+      schema: pevr_seeds
+      column_types:
+        id_indicateur: nvarchar(8)
+        description_indicateur: nvarchar(max)      
+        cod_matiere: nvarchar(32)
+        des_matiere: nvarchar(32)
+  
+  - name: custom_indicateurs_pevr
```

Comment:
@sadqim perso j'ajouterais des tests (unique, not null). C"est une table publique. Il faut tester le plus bas possible pour retourner des erreurs qui fassent écho chez l'intégrateur. Plus de probabilité qu'il comprenne si l'erreur remontée est proche de son code

Reply:
oui tu as raison